app/src/main/java/edu/grinnell/grinnell_publications_android/Services/POJO/JStory.java

```diff
@@ -0,0 +1,110 @@
+package edu.grinnell.grinnell_publications_android.Services.POJO;
+
+
+import java.util.List;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class JStory {
```

Comment:
What is a JStory? I'm still trying to figure out what this is. The class name is not as descriptive.